otter/convergence/errors.py

```diff
@@ -43,6 +47,11 @@ def _present_clb_deleted_error(exception):
             % (exception.lb_id,))
 
 
+@_present_exception.register(CreateServerConfigurationError)
+def _present_server_configuration_error(exception):
+    return "Server launch configuration is invalid: %s" % (exception.message,)
```

Comment:
I was mostly just following the previous convention of the other `_present_server_configuration_error` things.  I generally prefer `format`, but I wasn't sure if we still follow that.  I'll be happy to change both this and all the rest to use format instead :)